### PR TITLE
feature(docs)  add consumer route

### PR DESCRIPTION
**Pull request by olmstead**
This route will return the root level unauthenticated application data which includes the web appearance, firm info, feature flags, and questionnaire questions.

  Closes: CU2-142

@eweise @antoinemeyer @johntowell @abuecker 
ready for review

### CODE REVIEW

File: api/v3/examples/consumer.res.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    "embeddableCompanyVideo": "http://www.youtube.com/embed/7NQ9OS6biM8?modestbranding=1&autoplay=1&rel=0",
+    "headerColor": "#000111",
+    "primarySiteColor": "#e98791",
+    "primaryButtonColor": "#cc5e0e",
+    "getStartedButtonColor": "#ED6F00",
+    "alternateFontColor": "#EEEEEE",
+    "progressColor": "#ABCDEF",
+    "header": "this is my header",
+    "firmWebAppearanceTeamBiosCollection":[
+      {
```
Comment (olmstead) [inline api/v3/examples/consumer.res.json]: This may no longer be a requirement, but existed in the old website route.

**Comment by olmstead** [comment]
@bump

**Comment by johntowell** [comment]
I think we should have ```"additionalProperties": false``` for all our response schemas. Since we are validating in the test this will keep our code and schema in sync. Otherwise we can return anything.

**Comment by olmstead** [comment]
Added **"additionalProperties": false** to all definitions.

**Comment by johntowell** [comment]
👍

**Comment by olmstead** [comment]
On implementation, I realized we want to prune out some of the web appearances. Making some changes to this PR. Stand by.

**Comment by olmstead** [comment]
@abuecker @johntowell @eweise @antoinemeyer Ready for review
I a significant reworking of this based on my findings in the implementation side.  
* change **appearance** to more general **firmInfo**
* remove any unneeded attributes
* eliminate second request for **about us** and **investment philosophy** content.  all returned here instead.

**Comment by johntowell** [comment]
👍

**Pull request merged**